src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,116 @@
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final int ID_INDEX = 1;
+    private static final int NAME_INDEX = 2;
+    private static final int COUNTRY_INDEX = 3;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturerRequest = "INSERT INTO manufacturers(name,country) values(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                  PreparedStatement createManufacturerStatement =
+                          connection.prepareStatement(insertManufacturerRequest,
+                             Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerStatement.setString(1, manufacturer.getName());
+            createManufacturerStatement.setString(2, manufacturer.getCountry());
+            createManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                manufacturer.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufactured: "
+                    + manufacturer + " to DB", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT * FROM manufacturers where id = ? and is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                  PreparedStatement getManufacturerById
+                          = connection.prepareStatement(query)) {
+            getManufacturerById.setLong(1, id);
+            ResultSet resultSet = getManufacturerById.executeQuery();
+            Manufacturer manufacturer = null;
+            if (resultSet.next()) {
+                manufacturer = getManufacture(resultSet);
+            }
+            return Optional.ofNullable(manufacturer);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturer from DB", e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String query = "SELECT * FROM manufacturers where is_deleted = false;";
+        List<Manufacturer> allManufacturer = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                  Statement getAllManufacturer = connection.createStatement()) {
+            ResultSet resultSet = getAllManufacturer
+                    .executeQuery(query);
```

Comment:
```suggestion
            ResultSet resultSet = getAllManufacturer.executeQuery(query);
```
only in case it's no longer than 80 symbols